src/main/java/org/springframework/data/redis/cache/RedisCacheConfiguration.java

```diff
@@ -47,9 +47,11 @@
 public class RedisCacheConfiguration {
 
 	protected static final boolean DEFAULT_CACHE_NULL_VALUES = true;
+	protected static final boolean DEFAULT_ENABLE_TTI_EXPIRATION = false;
```

Comment:
Let's use `TIME_TO_IDLE` instead of `TTI` for more expressivity.

Reply:
Will do.

Ironically, I initially spelled it out in favor of being explicit, but ultimately opted to match our use of the "TTL" abbreviation.